docs/sgp40_voc_index/arduino.rst

```diff
@@ -0,0 +1,110 @@
+for SCL to connect the SCL pin, GND for the GND pin and a 3.3v supplie for the VCC pin. 
+Since we use the Arduino NANO 33 BLE as an example here a pinout of it with the correct 
+pins marked:
+
+|Arduino Pinout|
+
+Flashing Arduino Board
+----------------------
+
+Start by downloading the `sgp40-arduino-VERSION.zip` release form the Sensirion Github 
+page (https://github.com/Sensirion/embedded-sgp/releases).
+Unpack the `.zip` file and navigate into the resulting directory. Click on the `.ino` file, 
+this should start the Arduino IDE and open the project. If it doesn't work start the Arduino
+IDE, in the top bar click on File, Open... , navigate to the unziped folder and open the 
+`.ino` file. 
+
+For most 3.3v boards you need to install an additional package to be able to flash them via the 
+Arduino IDE. This is done selecting Tools, then Boards and last Boards Manager.
+Then you can type the name of your board in the search bar and install the upcoming package.
+
+|Boardmanager|
+
+Now you only need to upload the code to your board. Connect your Arduino board with your computer
+using an usb cable. Then you'll need to select the entry in the Tools, Board menu that corresponds 
+to your Arduino board. Finaly you also need to select the right port where your Arduino is conected. 
+For this navigate to Tools, Port and select the one inicating that your board is connected there
+(name of your board at the end in brackets). If no port is indicating that a board is connected 
+it's likely to be COM3. To now upload your code to your board click the arrow icon in the top lefthand corner. 
+This process can take a while. 
+
+Checkout Sensor Data
+--------------------
+
+To look at the senor data open the serial monitor by navigating to Tools, Serial Monitor. 
+If everything worked fine and your Arduino is still connected you now should see the messuring 
+data of the Sensor. 
+
+::
+
+    VOC Index: 132
+    Temperature: 28.41degC
+    Relative Humidity: 40.68RH
+    VOC Index: 129
+    Temperature: 28.32degC
+    Relative Humidity: 40.54RH
+    VOC Index: 127
+    Temperature: 28.25degC
+    Relative Humidity: 40.59RH
+    VOC Index: 125
+    Temperature: 28.30degC
+    Relative Humidity: 40.89RH
+    VOC Index: 124
+    Temperature: 28.33degC
+    Relative Humidity: 41.57RH
+    VOC Index: 124
+    Temperature: 28.35degC
+    Relative Humidity: 42.17RH
+    VOC Index: 124
+    Temperature: 28.26degC
+    Relative Humidity: 41.92RH
+    ...
+
+Troubleshooting
+---------------
+
+Initialization failed
+~~~~~~~~~~~~~~~~~~~~~
+
+-  Check if you connected the sensor correctly, so all cables are fully
+   pluged in and connected to the correct header.
+-  Check if you have enabled I2C on the Raspberry Pi. For this redo the
+   steps on "Enable the I2C interface in the Raspberry Pi config" in the
+   guide above.
+
+VOC Index is zero
+~~~~~~~~~~~~~~~~~
+
+For the first 100 iterations it is always zero, so just wait a little. 
```

Comment:
Explain why, people will be curious to know.

```suggestion
During the initialization phase of the VOC Index algorithm, which is the first 100 iterations, the VOC Index is hardcoded to zero.
```

Reply:
I did know either ...